src/main/java/Objects/NetworkDevices/PC.java

```diff
@@ -1,8 +1,40 @@
 package Objects.NetworkDevices;
 
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A specialized NetworkDevice that will
+ * serve as the mainstay of the player's
+ * botnet.  Capable of sending out all types
+ * of packets.
+ */
 public class PC extends NetworkDevice {
 
-    private String ipAddress;
-    //TODO: connected devices?
+    /**
+     * Constructor for a PC
+     * @param speed         Packet generation speed
+     * @param team          Team device belongs to
+     * @param connections   List of connected devices
+     * @param hidden        Whether device is hidden by fog of war
+     * @param max_packet    Max number of packets device can hold
+     * @param id            Unique device ID value
+     */
+    public PC(final int speed, final String team,
```

Comment:
it looks like a lot of these constructors are setting the same variables, why not implement a constructor in NetworkDevice to set them?

Reply:
Style issue.
Resolving.